### DISCUSSION TITLE
Parse time series

### DISCUSSION THREAD

**Discussion by Taoye2022** [torresolmx/PhoenixGeoPy#2] [Q&A]
Hi, Can I use this project to parse bin time series files and export them into ASCII files used for Joint time-frequency analysis？

**Comment by torresolmx** [comment]
Hello,

Sorry for the late reply.

The project is a library. It can be used to convert the time series into
numpy arrays. These arrays can then be exported to any form (you would need
to write the exporter desired).

I hope this helps.

Best regards,
Jorge

**Comment by Taoye2022** [comment]
Thank you so much.

Cheers,
Tao

On 2022-09-23 04:48, torresolmx wrote:
> Hello,
> 
> Sorry for the late reply.
> 
> The project is a library. It can be used to convert the time series
> into
> numpy arrays. These arrays can then be exported to any form (you would
> need
> to write the exporter desired).
> 
> I hope this helps.
> 
> Best regards,
> Jorge
> 
> On Fri., Jul. 1, 2022, 9:53 a.m. Taoye2022, ***@***.***>
> wrote:
> 
>> Hi, Can I use this project to parse bin time series files and export
> them
>> into ASCII files used for Joint time-frequency analysis？
>> 
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/torresolmx/PhoenixGeoPy/discussions/2>, or
> unsubscribe
>> 
> <https://github.com/notifications/unsubscribe-auth/AESGAQ7MKFV2X2R5F3YFWCTVR3Z7BANCNFSM52MZ5SEA>
>> .
>> You are receiving this because you are subscribed to this
> thread.Message
>> ID: ***@***.***>
>> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe
> [2].
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> 
> 
> Links:
> ------
> [1] 
> https://github.com/torresolmx/PhoenixGeoPy/discussions/2#discussioncomment-3713951
> [2] 
> https://github.com/notifications/unsubscribe-auth/AZ3PIVUXGSKQBDHNCFCAKPLV7UR7LANCNFSM52MZ5SEA

-- 
Tao Ye 
Post-doctoral Research Fellow
School of Cosmic Physics - Geophysics
Dublin Institute for Advanced Studies
5 Merrion Square North, Dublin 2 - D02 Y006
Tel:(353)872955543
www.dias.ie
#DIASdiscover
